command-line/dot-profile.md

```diff
@@ -108,11 +108,7 @@ of helpful libraries are written using scripts in this way.
 
 #### Aliases
 
-It's often helpful to make commands for yourself that are short cuts. Bash aliases allows you to set a shortcut command for a longer command. For example what if you want to change to your directory but you don't want to type 
- ```bash
- cd ~/scripts
- ```
-What if you could just type `cdscr` instead? Aliases are defined in the .bash_profile or .bashrc file typically in your home directory. 
+It's often helpful to make commands for yourself that are short cuts. Bash aliases allows you to set a shortcut command for a longer command. For example what if you want to change to your directory but you don't want to type `cd ~/scripts`. What if you could just type `cdscr` instead? Aliases are defined in the .bash_profile or .bashrc file typically in your home directory. 
```

Comment:
Why did this get changed?

Reply:
Hi Alina, it had a merge conflict with the master branch. But I've resolved it 
Nothing got charged